### DISCUSSION TITLE
Import not working

### DISCUSSION THREAD

**Discussion by tkallady** [florimondmanca/djangorestframework-api-key#228] [Q&A]
**Describe the bug**
rest_framework_api_key import is missing
python manage.py runserver still works,
gunicorn wsgi.py throws "ModuleNotFoundError: No module named 'rest_framework_api_key'"

**To Reproduce**
Create a basic new project with a view that contains the line 'from rest_framework_api_key.permissions import HasAPIKey'

**Expected behavior**
Should import module without error.

**Screenshots**
![image](https://user-images.githubusercontent.com/43975406/197093690-f6aa4b5c-1dd9-4fd0-8aef-b9e9acbbaee5.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Python Version: 3.8
 - Django Version: 4.07
 - DRF Version: 3.14.0

**Comment by florimondmanca** [comment]
@tkallady Hi, it sounds like you're using a Python interpreter different than the one where the dependency was installed. I'd recommend checking with `python -c "import rest_framework_api_key"`.
